toonz/sources/CMakeLists.txt

```diff
@@ -187,7 +187,7 @@ endif()
 include_directories(BEFORE ${TIFF_INCLUDE_DIR})
 
 find_path(BOOST_ROOT include/boost boost HINTS ${THIRDPARTY_LIBS_HINTS} PATH_SUFFIXES boost155/1.55.0_1 boost/boost_1_55_0/)
-find_package(Boost 1.55 EXACT REQUIRED)
+find_package(Boost 1.55 REQUIRED)
```

Comment:
Ahhh, interesting.  Was wondering if newer Boost would work, and haven't bothered trying yet.